### PR TITLE
Use assertThatThrownBy in TestSetSessionTask

### LINKED ISSUE #6082: Use assertThatThrownBy in TestSetSessionTask
Make tests in `TestSetSessionTask` more readable by using `assertThatThrownBy`.

e.g. https://github.com/prestosql/presto/blob/master/presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java#L155

### PR DESCRIPTION
**Pull request by somayaj**
Fixes #6082

The old PR is https://github.com/prestosql/presto/pull/6128

### CODE REVIEW

**Review by ebyhr — APPROVED** [review]

File: presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java [hunk 1]
```diff
@@ -57,6 +57,7 @@
 import static java.lang.String.format;
 import static java.util.Collections.emptyList;
 import static java.util.concurrent.Executors.newCachedThreadPool;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.fail;
```
Comment (ebyhr) [inline presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java:62]: Please remove this unused import.

Comment (somayaj) [reply presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java]: done

Comment (ebyhr) [reply presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java]: @somayaj Thanks, please squash commits into one.

Comment (somayaj) [reply presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java]: done

**Comment by ebyhr** [comment]
@cla-bot check

**Comment by ebyhr** [comment]
@cla-bot check

**Comment by somayaj** [comment]
Hi is this merged yet?

**Comment by losipiuk** [comment]
@cla-bot check

**Comment by losipiuk** [comment]
@somayaj thanks for contribution. DId you submit your CLA already?

**Comment by somayaj** [comment]
yes I did

**Comment by losipiuk** [comment]
>> @somayaj thanks for contribution. DId you submit your CLA already?

> yes I did

@martint can you take a look? The CLA bot seems to have other opinion on that  :)

**Comment by ebyhr** [comment]
@cla-bot check

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**